pkg/provider/aws/action/openshift-snc/openshift-snc.go

```diff
@@ -261,18 +297,31 @@ func manageResults(stackResult auto.UpResult, prefix *string) error {
 		fmt.Sprintf("%s-%s", *prefix, outputKubeAdminPass): "kubeadmin_pass",
 		fmt.Sprintf("%s-%s", *prefix, outputDeveloperPass): "developer_pass",
 	}
-	if err := output.Write(
-		stackResult, maptContext.GetResultsOutputPath(), results); err != nil {
-		return err
+
+	outputPath := maptContext.GetResultsOutputPath()
+	if outputPath == "" {
```

Comment:
recommended here is if len(maptContext.GetResultsOutputPath()) == 0

Actually I thought it when reviewing kind metadata but forgot to add it there too